python/deltalake/writer.py

```diff
@@ -44,6 +44,9 @@ def write_deltalake(
     partition_by: Optional[List[str]] = None,
     filesystem: Optional[pa_fs.FileSystem] = None,
     mode: Literal["error", "append", "overwrite", "ignore"] = "error",
+    name: Optional[str] = None,
```

Comment:
Could we also add a note in the docstring that this function does *not* register this table in your data catalog? Users will have to either use the filesystem path to the table or manually register this table in the catalog. I'll create a follow up issue for us to handle that though.

Reply:
Added a brief note. Let me know if more explanation re: manual add is needed.